apps/easypid/src/app/(app)/_layout.tsx

```diff
@@ -172,7 +172,9 @@ export default function AppLayout() {
           <Stack.Screen name="activity/[id]" options={headerNormalOptions} />
           <Stack.Screen name="pinConfirmation" options={headerNormalOptions} />
           <Stack.Screen name="pinLocked" options={headerNormalOptions} />
-          <Stack.Screen name="federation" options={headerNormalOptions} />
+          <Stack.Screen name="trust/federation" options={headerNormalOptions} />
```

Comment:
They just use the same screen, but I split it up so it might be better in the future to have custom UI for this.